reactioncommerce/reaction
# Remove legacy PDP

Resolves #2080.

How to Test:
* Run the app and visit the PDP.
* Check that the PDP is working as it should (using React components)
* Check with an old revision (since this is removing the isRevisionControlEnabled check).
